internal/schema/ize-spec.json

```diff
@@ -137,6 +137,9 @@
                         "version": {
                             "type": "string"
                         },
+                        "terraform_version": {
+                            "type": "string"
```

Comment:
Can we make sure this is deprecated if used within [infra] section?

Reply:
@AutomationD In this [PR](https://github.com/hazelops/ize/pull/402) added deprecation message